shared/actions/chat2/index.tsx

```diff
@@ -1786,6 +1786,31 @@ function* messageSend(state: TypedState, action: Chat2Gen.MessageSendPayload, lo
   // narrow down the places where the action can possibly stop.
   logger.info('non-empty text?', text.stringValue().length > 0)
 }
+const messageSendByUsername = async (state: TypedState, action: Chat2Gen.MessageSendByUsernamePayload) => {
+  const tlfName = `${state.config.username},${action.payload.username}`
+  const result = await RPCChatTypes.localNewConversationLocalRpcPromise(
```

Comment:
This can throw errors, you probably want to try this on a bad network, right now its just going to black bar.